src/Microsoft.Diagnostics.Monitoring.RestServer/Controllers/DiagController.cs

```diff
@@ -21,46 +27,100 @@ public DiagController(ILogger<DiagController> logger, IDiagnosticServices diagno
             _diagnosticServices = diagnosticServices;
         }
 
-        [HttpGet("getpids")]
-        public ActionResult<IEnumerable<string>> GetPids()
+        [HttpGet("processes")]
```

Comment:
>processes [](start = 18, length = 9)

Don't we want the versioning (as an optional part of the path)?

Reply:
We do, but we don't have different versions yet. Also, I don't know if this has been discussed enough, for example, it each api route versioned independently or is the entire api surface versioned all together?

Reply:
I'm not sure to be honest. We can table this for now, if it's easier.